src/monospace/index.ts

```diff
@@ -72,7 +72,8 @@ async function pollAuthorizedProject(rid: string): Promise<string | null> {
     return pollAuthorizedProject(rid);
   }
 
-  // FIXME: This is not being reached as the process exits before a new call is made
+  // TODO: Review this. It's not being reached as the process exits before a new
```

Comment:
If this isn't being reached, what happens when the user cancels the dialog and how is that handled?

Reply:
It's being handled in this block as `userResponse.success` is coming as `false`: https://github.com/firebase/firebase-tools/blob/556df35b0532bebefedc94c595b8272b92ee5a80/src/monospace/index.ts#L57-L59


in the tests I did I never received `error` as `USER_CANCELED`